package/src/components/Message/utils/messageActions.ts

```diff
@@ -86,7 +87,7 @@ export const messageActions = <
     actions.push(editMessage);
   }
 
-  if (copyMessage !== undefined && message.text && !error) {
+  if (setClipboardString !== null && message.text && !error) {
```

Comment:
Do we need to explicitly check against `null` here ? Otherwise, simply doing 
```suggestion
  if (setClipboardString && message.text && !error) {
```
should be enough.

Reply:
Yes, since the native.ts checks for `undefined`. And if it's not undefined, we set the function to null. Else `fail` is set which would throw the error.